client/hotrod-client/src/test/java/org/infinispan/client/hotrod/event/ClientListenerRemoveOnStopTest.java

```diff
@@ -30,4 +32,24 @@ public void testAllListenersRemovedOnStop() {
       assertEquals(0, listeners.size());
    }
 
+   public void testRemoveListenerAfterStopAndRestart() {
+      final RemoteCache<Integer, String> rcache = remoteCacheManager.getCache();
+      final EventLogListener<Integer> eventListener1 = new EventLogListener<>(rcache);
+      rcache.addClientListener(eventListener1);
+      Set<Object> listeners = rcache.getListeners();
+      assertEquals(1, listeners.size());
+      assertEquals(eventListener1, listeners.iterator().next());
+      listeners = rcache.getListeners();
+      assertEquals(1, listeners.size());
```

Comment:
forfot to delete the lines ! test above have 2 listeners but for the test I've done proving the bug I only needed one ! I deleted the double check